test/geo.test.js

```diff
@@ -131,3 +133,96 @@ describe('GeoPoint', function() {
     });
   });
 });
+
+describe('nearFilter', function() {
+  it('should handle an empty where gracefully', function() {
+    var result = nearFilter(null);
+    result.should.be.false;
+  });
+
+  it('should handle a simple where filter with location', function() {
```

Comment:
Please change the test description accordingly in all places based on this comment: https://github.com/strongloop/loopback-datasource-juggler/pull/1028/files#r73892175